WeWatch/WeWatch/ContentView.swift

```diff
@@ -26,6 +28,16 @@ internal struct ContentView: View {
                 Text(status)
                     .font(.poppinsRegular16px)
                     .foregroundStyle(.green)
+                Text(viewModel.errorMessage ?? "")
+                let date = Date(timeIntervalSince1970: TimeInterval(expireTime))
+                if date > Date() {
+                    
+                    Text("Result: \(date)")
+                        .foregroundColor(.green)
+                } else {
+                    Text("Valid time: \(date)")
+                        .foregroundColor(.red)
+                }
```

Comment:
if viewModel.isValidToken { 
     Text("Result: \()")
                  .foregroundColor(.green)
} else { 
       Text("Token is expired: \(date)")
                        .foregroundColor(.red)
}